.travis.yml

```diff
@@ -1,12 +1,16 @@
+sudo: false
+
 language: node_js
+
 node_js:
-  - '0.10'
```

Comment:
Its about to be deprecated. But on second thought, lets add it because a number of people still use 0.xx.